react/FilterNavigator.js

```diff
@@ -94,8 +95,9 @@ const FilterNavigator = ({
 
   const mobileLayout =
     (isMobile && layout === LAYOUT_TYPES.responsive) ||
-    layout === LAYOUT_TYPES.mobile
-
+    layout === LAYOUT_TYPES.mobile ||
+    !isMobile && layout === LAYOUT_TYPES.phone || isMobile && layout === LAYOUT_TYPES.phone
```

Comment:
I think you can simplify it like this:
```suggestion
    layout === LAYOUT_TYPES.phone
```

Reply:
I agree! You should do that.

Reply:
Done. Thanks for suggestions :D